e2e/cli-e2e/tests/collect.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('CLI collect', () => {
   const exampleAuditTitle =
     'Require `const` declarations for variables that are never reassigned after declared';
 
+  console.log('hello');
```

Comment:
Looks like you've forgot to remove the log

Reply:
This was added on purpose in order for nx to resolve affected projects and run some tests. By default, no projects were matched as the .github folder does not belong in any package.